docs/index.md

```diff
@@ -16,57 +16,78 @@ OpenVINO™ toolkit:
 The following diagram illustrates the typical OpenVINO™ workflow (click to see the full-size image):
 ![](img/OpenVINO-diagram.png)
 
-### Model Preparation, Conversion and Optimization
+### Model Preparation
+#### Components: [Open Model Zoo](https://github.com/opencv/open_model_zoo), [Deep Learning Workbench](@ref workbench_docs_Workbench_DG_Introduction), [OpenVINO Training Extentions](https://github.com/openvinotoolkit/training_extensions)
 
 You can use your framework of choice to prepare and train a deep learning model or just download a pre-trained model from the Open Model Zoo. The Open Model Zoo includes deep learning solutions to a variety of vision problems, including object recognition, face recognition, pose estimation, text detection, and action recognition, at a range of measured complexities.
-Several of these pre-trained models are used also in the [code samples](IE_DG/Samples_Overview.md) and [application demos](@ref omz_demos_README). To download models from the Open Model Zoo, the [Model Downloader](@ref omz_tools_downloader_README) tool is used.
+Several of these pre-trained models are used also in the [code samples](IE_DG/Samples_Overview.md) and [application demos](@ref omz_demos_README). To download models from the Open Model Zoo, the [Model Downloader](@ref omz_tools_downloader_README) tool is used. 
 
-One of the core component of the OpenVINO™ toolkit is the [Model Optimizer](MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md) a cross-platform command-line
+[Deep Learning Workbench](@ref workbench_docs_Workbench_DG_Introduction) provides a we-based interface to download a pre-trained model and work with it. 
```

Comment:
a web-based (correct typo)

Reply:
Better to extend the sentence with explanation what does it mean 'work with it'. Instead of 'work with it' put : enables you to visualize, fine-tune, and compare performance of deep learning models

Reply:
Agree, I've applied these comments.